autogen/agentchat/conversable_agent.py

```diff
@@ -1984,7 +2005,9 @@ def get_human_input(self, prompt: str) -> str:
         Returns:
             str: human input.
         """
-        reply = input(prompt)
+        iostream = IOStream.get_default()
```

Comment:
How does a web application pass human input into the conversable agent?

Reply:
So when `human_input_model="ALWAYS"`, each time human is prompt for input, the server side would wait on the websocket for human inputs coming in from the client side?

Reply:
yes